docs/config/drivers.md

```diff
@@ -125,6 +125,19 @@ browsers: [{
 }]
 ```
 
+## Microsoft Edge (based on Chromium)
+Microsoft Edge Driver is downloadable separately from Windows. Please make sure you have the correct version installed as explained in [Microsoft Edge](https://developer.microsoft.com/en-us/microsoft-edge/tools/webdriver/). The WebDriver release version should match browser version, for example, for Microsoft Edge (based on Chromium) Release 79, choose WebDriver Release 79. The downloaded driver should be moved to <uiveri5-installation-folder>/selenium/ without renaming.
```

Comment:
autoupdate

Reply:
autoupdate is working based on driverVersions.json